### PR TITLE
Update concourse pipelines for Control Panel

### PR DESCRIPTION
**Pull request by andyhd**

### CODE REVIEW

File: alpha/cpanel-api.yaml [hunk 1]
```diff
@@ -76,4 +76,4 @@ resource_types:
   type: docker-image
   source:
     repository: linkyard/concourse-helm-resource
-    tag: 2.9.1
+    tag: latest
```
Comment (r4vi) [inline alpha/cpanel-api.yaml]: are we sure we want `latest`?

**Review by r4vi — APPROVED** [review]
I'm guessing the override values changed because the latest image uses different names?

**Comment by andyhd** [comment]
Yes, I renamed the values in the Helm chart, but the overrides here were unchanged

File: alpha/cpanel-api.yaml (hunk 1 above)
Comment (andyhd) [reply alpha/cpanel-api.yaml]: They don't do a `2.9` tag to catch all the minor releases. Do we need the helm client to be the same version or lower than Tiller?

Comment (andyhd) [reply alpha/cpanel-api.yaml]: Or `2.14`, or whatever

Comment (andyhd) [reply alpha/cpanel-api.yaml]: Changed it to match current server version

**Pull request merged**